core/src/main/java/feign/EncodeSlash.java

```diff
@@ -0,0 +1,13 @@
+package feign;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target(METHOD)
+@Retention(RUNTIME)
+public @interface EncodeSlash {
```

Comment:
Slightly misleading, since it isn't actually encoding the slash, it's just not decoding it.  Should it be a full annotation?  How about just an attribute on `RequestLine`?